src/components/sections/unlock.tsx

```diff
@@ -7,7 +7,7 @@ export default function Unlock() {
     <>
 
     {/* mobile Version */}
-    <div className=" hidden lg:inline-flex  w-[100%] border h-auto flex-col justify-start items-center gap-[236px] inline-flex bg-cover bg-no-repeat bg-center bg-unlock  bg-[url('../../public/unlock.png')]">
+    <div className=" hidden lg:inline-flex  w-[100%]  h-auto flex-col justify-start items-center gap-[236px] inline-flex bg-cover bg-no-repeat bg-center bg-unlock  bg-[url('../../public/unlock.png')]">
```

Comment:
update the public path using next js conventions,
 and the public folder is mounted on the open directory
 no need to use ../../../